.github/ISSUE_TEMPLATE/issue-report.yml

```diff
@@ -0,0 +1,163 @@
+  - type: textarea
+    id: environment-information
+    attributes:
+      label: Environment
+      description: |
+        Accurately fill in as much environment details as possible. If a certain environment field is not shown in the template below, but you consider useful information, please include it.
+        Examples:
+        - **OS**: [e.g. Debian 12, Windows 11]
+        - **Linux Kernel**: [e.g. none, 5.15, 6.1, etc.]
+        - **Virtualization**: [e.g. Docker, KVM, LXC]
+        - **Browser**: [e.g. Firefox 91, Chrome 93, Safari 13, native client]
+        - **Installed Plugins**: [...]
+        - **Reverse Proxy**: [e.g. none, nginx, apache, cloudflared, etc.]
+        - **Base URL**: [e.g. none, yes: /example]
+        - **Networking**: [e.g. Host, Bridge/NAT]
+        - **Storage**: [e.g. local, NFS, cloud]
+      value: |
+        - OS:
+        - Linux Kernel:
+        - Virtualization:
+        - Browser:
+        - Plugins:
+        - Reverse Proxy:
+        - Base URL:
+        - Networking:
+        - Storage:
+      render: markdown
+    validations:
+      required: true
+  - type: markdown
+    id: general-information-logs
+    attributes:
+      value: |
+        When providing logs, please keep the following things in mind:
+        1. **DO NOT** use external paste services. If logs are too large to paste into the field, upload them as text files.
+        2. Please provide complete logs.
+          - Ensure to capture all relevant information, encompassing both the events leading up to and following the occurrence of the issue. Typically, providing 10 *lines preceding and succeeding* the problem should be adequate.
+        3. Please do not run logs through any translation program. We exclusively accept raw, untranslated logs. Particularly exercise caution if your browser automatically translates pages by default.
+          - **DO NOT** forget to censor out personal information such as public IP addresses.
+        4. Please do not include logs as screenshots, with the only exception being client logs in browsers.
+  - type: textarea
+    id: openbullet-logs
+    attributes:
+      label: OpenBullet2 logs
+      description: Please copy and paste any relevant log output.
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: browser-logs
+    attributes:
+      label: Client / Browser logs
+      description: Access browser logs by using the F12 to bring up the console. Screenshots are typically easier to read than raw logs.
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Relevant screenshots or videos
+      description: Attach relevant screenshots or videos related to this report.
+  - type: textarea
+    id: lolicode
+    attributes:
+      label: Relevant LoliCode if needed
+      description: Please copy and paste any relevant LoliCode to reproduce the issue. This will be automatically formatted into code, so no need for backticks.
+      render: shell
```

Comment:
I would render this as `csharp` since LoliCode can have C# code inside, so it's the closest highlighting language